File: Editorial/220_Contains_Duplicate_III/CPP/1_Approach_1_Time_Limit_Exceeded.cpp
class Solution {
public:
    bool containsNearbyAlmostDuplicate(vector<int>& nums, int k, int t) {
        for (int i = 0; i < nums.size(); ++i) {
            for (int j = max(i - k, 0); j < i; ++j) {
                if (abs((long long)nums[i] - nums[j]) <= t) return true;
            }
        }
        return false;
    }
};

File: Editorial/0220_Contains_Duplicate_III/Python3/1_Approach_1_Time_Limit_Exceeded.py
class Solution:
    def containsNearbyAlmostDuplicate(
        self, nums: List[int], k: int, t: int
    ) -> bool:
        for i in range(len(nums)):
            for j in range(max(i - k, 0), i):
                if abs(nums[i] - nums[j]) <= t:
                    return True
        return False
